style.css

```diff
@@ -363,45 +272,381 @@ li {
   margin: 0.5px;
 }
+
+@media screen and (max-width: 767px) {
+  .logo {
+    padding: 20px;
+    color: #fff;
+    font-style: normal;
+    text-decoration: none;
+  }
+
+  .button {
+    display: block;
+    padding: 20px;
+  }
+
+  .options {
+    display: none;
+  }
+
+  .main-class {
+    height: 560px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 0 20px;
+    background-color: #1c1a19;
+    background-image: url("./ressources/Ilustration/illustration-mobile.png");
+    background-repeat: no-repeat;
+    background-position: right bottom;
+  }
+
+  .description {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    width: 80%;
+  }
+
+  .icons {
+    display: flex;
+  }
+
+  .works {
+    display: grid;
+    margin: auto;
+    width: 327px;
+  }
+
+  .head-works {
+    font-family: "Crete Round", serif;
+    font-size: 40px;
+    font-weight: 400;
+    text-align: center;
+    padding: 1% 6.3%;
+    margin-top: 30%;
+  }
+
+  .content {
+    margin-top: 50px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .project img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 8%;
+    width: 300px;
+  }
+
+  .content1 {
+    background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.103) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("ressources/Ilustration/Prj1.png");
+    background-size: cover;
+    background-position: center;
+    padding: 21% 3%;
+    position: relative;
+    margin-top: 10%;
+    margin-bottom: 8%;
+    height: 386px;
+    width: 327px;
+  }
+
+  .content2 {
+    background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.103) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("ressources/Ilustration/Pr2.png");
+    background-size: cover;
+    background-position: center;
+    padding: 21% 3%;
+    position: relative;
+    margin-top: 10%;
+    margin-bottom: 8%;
+    height: 386px;
+    width: 327px;
+  }
+
+  .content3 {
+    background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.103) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("ressources/Ilustration/pr3.png");
+    background-size: cover;
+    background-position: center;
+    padding: 21% 3%;
+    position: relative;
+    margin-top: 10%;
+    margin-bottom: 8%;
+    height: 386px;
+    width: 327px;
+  }
+
+  .about-me {
+    background-color: #1c1a19;
+    flex-direction: column;
+    padding: 0.1%;
+    background-image: url("ressources/Ilustration/illustration-mobile1.png");
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    padding-bottom: 47%;
+  }
+
+  .skills {
+    text-align: center;
+    background-color: #1c1a19;
+    padding: 100px 6.4%;
+  }
+
+  .skills-title {
+    display: flex;
+    justify-content: center;
+    font-size: 32px;
+    margin-top: 70px;
+    color: #fff;
+    font-weight: 700;
+    font-family: "Inter", sans-serif;
```

Comment:
please kindly reduce the title of the skills to be similar to the Figma design we have 👍 

Your design :
![image](https://github.com/AndresEspi/portafolio/assets/20537712/e3fc3fb4-bf25-4a99-8da4-9c86253650d9)